checkbox/lib/checkbox.ts

```diff
@@ -110,6 +110,11 @@ export class Checkbox extends LitElement {
       this.prevDisabled = changed.get('disabled') ?? this.disabled;
       this.prevIndeterminate =
           changed.get('indeterminate') ?? this.indeterminate;
+      
+      this.dispatchEvent(new CustomEvent('checked', {
```

Comment:
Would the `input` event not be enough in this case?
```js
checked.addEventListener("input", updateValue);

function updateValue(e) {
  console(e.target.checked);
}```

Reply:
@christophe-g The problem is the same with `onChange`, if you change `checkbox.checked` manually it doesn't subsequently call `change` or `input`.
I don't think this is an issue because that's the native behavior with html native inputs, that's why I thought having a dedicated event (with detail value) for that would be handy.

Reply:
@vdegenne  - ah d'accord

I think the tendency here is to follow native behaviors - hence only dispatch events when user interact with the element (e.g. click or keyboard input).